### ISSUE TITLE
Why add entropy to loss when its gradient is zero?

### ISSUE THREAD

**Issue by Akella17** [ikostrikov/pytorch-a2c-ppo-acktr-gail#194]
As the differential entropy of a Gaussian distribution is dependent only on the standard deviation, how does adding it to the actor loss function promote exploration? Isn't the gradient of the differential entropy of a Gaussian distribution w.r.t policy parameters 0?

h(X) = ln[σ(2π)<sup>1/2</sup>] + 0.5
(σ is a constant vector of ones)

**Comment by ikostrikov** [comment]
In this implementation, a Gaussian distribution has a learnable standard deviation parameter:
https://github.com/ikostrikov/pytorch-a2c-ppo-acktr-gail/blob/master/a2c_ppo_acktr/distributions.py#L79
